### PR TITLE
[ADD] Modules on payslip reporting

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
# HR Payslip report

This module adds a new report in the area of Human Resources that makes it
possible to report on the employee payslips.
# HR Payslip account report

This module adds a new report in the area of Human Resources that makes it
possible to report on the journal items generated by employee payslips.
# HR Payslip account report hr period

This module extends the HR Payslip account report to add the HR Period.

### CODE REVIEW

File: hr_payslip_account_report/README.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Credits
+=======
+
+Contributors
+------------
+
+* Jordi Ballester <jordi.ballester@eficent.com>
+
+Maintainer
```
Comment (elicoidal) [inline hr_payslip_account_report/README.rst:33]: I miss the section Bug tracker

File: hr_payslip_account_report/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        "python": [],
+        "bin": [],
+    },
+    "depends": [
+        "hr_payroll_account",
+    ],
+    "data": [
+        "report/hr_payslip_account_report_view.xml",
+        "security/ir.model.access.csv"
+    ],
```
Comment (elicoidal) [inline hr_payslip_account_report/__openerp__.py:20]: Please remove empty keys

File: hr_payslip_account_report_hr_period/README.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Go to 'Reporting | Human Resources | Payslip Account Report'
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/116/7.0
+
+
+Credits
```
Comment (elicoidal) [inline hr_payslip_account_report_hr_period/README.rst:25]: Please add the bug tracker section

File: hr_payslip_account_report_hr_period/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        "python": [],
+        "bin": [],
+    },
+    "depends": [
+        "hr_payslip_account_report",
+        "hr_period"
+    ],
+    "data": [
+        "report/hr_payslip_account_report_view.xml",
+    ],
```
Comment (elicoidal) [inline hr_payslip_account_report_hr_period/__openerp__.py:20]: Please remove empty keys

**Comment by JordiBForgeFlow** [comment]
Thanks @elicoidal for your review. Submitted new commit with the corrections.

**Comment by elicoidal** [comment]
Thanks for your contribution!

**Comment by JordiBForgeFlow** [comment]
Travis error are not related to the modules being added.

**Pull request merged**